packages/kbn-xstate-utils/src/console_inspector.ts

```diff
@@ -44,7 +44,7 @@ export const createConsoleInspector = () => {
         '🔔 %c%s%c received event %c%s%c from %c%s%c:',
         ...styleAsActor(eventEvent.actorRef.id),
         ...styleAsKeyword(eventEvent.event.type),
-        ...styleAsKeyword(eventEvent.sourceRef?.id),
+        ...styleAsKeyword(eventEvent.sourceRef?.sessionId),
```

Comment:
@weltenwort jfyi, had to backport a part of your change from https://github.com/elastic/kibana/pull/211784 here, hope that's fine.

Reply:
sure, I don't see why not - thanks for taking care of it!